Pristine-1.3: Two warnings in rmt.c

This was probably introduced with the authentication stuff.

```
 CC [M]  net/rina/rmt.o
net/rina/rmt.c: In function ‘n1_port_write_noclean’:
net/rina/rmt.c:688:32: warning: unused variable ‘ret’ [-Wunused-variable]
         int                    ret = 0;
                                ^
net/rina/rmt.c: In function ‘extract_policy_parameters’:
net/rina/rmt.c:552:3: warning: ignoring return value of ‘kstrtouint’, declared with attribute warn_unused_result [-Wunused-result]
   kstrtouint(policy_param_value(parameter), 10, &entry->initial_ttl_value);
```

--- a/linux/net/rina/rmt.c
+++ b/linux/net/rina/rmt.c
@@ -549,7 +549,13 @@ static int extract_policy_parameters(struct dup_config_entry * entry)
 			return -1;
 		}
 
-		kstrtouint(policy_param_value(parameter), 10, &entry->initial_ttl_value);
+		if (kstrtouint(policy_param_value(parameter),
+                               10,
+                               &entry->initial_ttl_value)) {
+                        LOG_ERR("Failed to convert TTL string to int");
+			return -1;
+                }
+
 		LOG_DBG("Initial TTL value is %u", entry->initial_ttl_value);
 	}
 
@@ -675,17 +681,16 @@ EXPORT_SYMBOL(rmt_config_set);
 
 static int n1_port_write_noclean(struct rmt *         rmt,
                                  struct rmt_n1_port * n1_port,
-                                 struct pdu * pdu)
+                                 struct pdu *         pdu)
 {
-        struct sdu *           sdu;
-        struct pdu_ser *       pdu_ser;
-        port_id_t              port_id;
-        struct buffer *        buffer;
-        struct ipcp_instance * n1_ipcp;
-        struct pci *           pci;
-        size_t                 ttl;
+        struct sdu *              sdu;
+        struct pdu_ser *          pdu_ser;
+        port_id_t                 port_id;
+        struct buffer *           buffer;
+        struct ipcp_instance *    n1_ipcp;
+        struct pci *              pci;
+        size_t                    ttl;
         struct dup_config_entry * dup_conf;
-        int                    ret = 0;
 
         ASSERT(n1_port);
         ASSERT(rmt);

